### PR TITLE
fix: Disable Select/Deselect all buttons when ui.dropdown is disabled

### LINKED ISSUE #1821: Select All | Deselect All buttons are functional when dropdown is disabled
#### Wave SDK Version, OS

Wave 0.24.2
Ubuntu 20.04

#### Actual behavior

When a ui.dropdown component is rendered with disabled=True, the "Select All | Deselect All" buttons are still enabled and functional (see the gif below)

#### Expected behavior

When disabled=True, the "Select All | Deselect All" buttons should get disabled.

#### Steps To Reproduce

Example of the code
```
ui.dropdown(
        name="cols",
        label="Columns to Drop",
        choices=[ui.choice(c, c) for c in cols],
        values=cols,
        required=True,
        trigger=False,
        disabled=True,
    )
```
dropdown_disabled

### PR DESCRIPTION
**Pull request by marek-mihok**
Closes #1821

### CODE REVIEW

**Comment by Far0n** [comment]
that was fast :D

File: ui/src/dropdown.test.tsx [hunk 1]
```diff
@@ -128,6 +128,14 @@ describe('Dropdown.tsx', () => {
       expect(wave.args[name]).toMatchObject(['A', 'B', 'D'])
     })
 
+    it('Do not select all options on Select all - dropdown disabled', () => {
+      const { getByText } = render(<XDropdown model={{ ...defaultProps, values: ['A'], disabled: true }} />)
+
+      fireEvent.click(getByText('Select All'))
+
+      expect(wave.args[name]).toMatchObject(['A'])
```
Comment (mturoci) [inline ui/src/dropdown.test.tsx:136]: Let's make the initial state clear.
```suggestion
      expect(wave.args[name]).toMatchObject(['A'])

      fireEvent.click(getByText('Select All'))

      expect(wave.args[name]).toMatchObject(['A'])
```

File: ui/src/dropdown.test.tsx [hunk 2]
```diff
@@ -146,6 +154,14 @@ describe('Dropdown.tsx', () => {
       expect(pushMock).toHaveBeenCalled()
     })
 
+    it('Do not deselect all options on Deselect all - dropdown disabled', () => {
```
Comment (mturoci) [inline ui/src/dropdown.test.tsx:157]: Same as comment above.

**Review by mturoci — CHANGES_REQUESTED** [review]
Thanks @marek-mihok! One small comment and it's good to go.

**Review by mturoci — APPROVED** [review]

**Pull request merged**